<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCasting</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLineEdit" name="file_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="file_open">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make_casting">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать кастинг</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="casting_text">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameCast">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name_c">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите название кастинга</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_instruct">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_one">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить человека</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
